Makefile

```diff
@@ -42,13 +42,19 @@ manager: fmt vet
 
 # Build the clusterctl binary
 clusterctl: fmt vet
-	go build -o bin/clusterctl ./cmd/clusterctl
+	CGO_ENABLED=0 go build -ldflags '-extldflags "-static" -w -s' \
+	-o bin/clusterctl."$${GOOS:-$$(go env GOHOSTOS)}"_"$${GOARCH:-$$(go env GOHOSTARCH)}" \
```

Comment:
The GOOS and GOHOSTARCH overrides doesn't seem necessary for this case?

Reply:
oh nvm this is just the bin name

Reply:
/hold

Reply:
Wait, never mind. Thought I saw a bug. There's not.

Reply:
/hold cancel